http-plugins/pom.xml

```diff
@@ -60,6 +56,28 @@
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>hydrator-test</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>io.cdap.cdap</groupId>
+          <artifactId>cdap-unit-test</artifactId>
+        </exclusion>
+        <!-- pulls in incompatible jackson versions due to avro -->
+        <exclusion>
```

Comment:
Is this exclusion and inclusion only for overwriting the jackson-databind version? If so, can we use dependencyManagement instead to set a desired version?

Reply:
oh I think I can remove this now since I had to add a direct dependency on it (tried doing this first).